src/tests/system/tests/test_passkey.py

```diff
@@ -592,3 +594,115 @@ def test_passkey__check_tgt(client: Client, ipa: IPA, moduledatadir: str, testda
 
     assert rc == 0, "Authentication failed"
     assert "Ticket cache" in output, "Failed to get the TGT"
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.builtwith(client="passkey", ipa="passkey")
+@pytest.mark.require.with_args(passkey_requires_root)
+def test_passkey__ipa_server_offline(
+    client: Client, ipa: IPA, moduledatadir: str, testdatadir: str, umockdev_ipaotpd_update
+):
+    """
+        :title: Check the authentication of user after kdestroy and when ipa service stop.
+    :setup:
+        1. Add a user with --user-auth-type=passkey in the server with passkey mapping.
+        2. Setup SSSD client with FIDO and umockdev, start SSSD service.
+    :steps:
+        1. Check authentication of the user and TGT after authentication.
+        2. Remove the tgt using #kdestroy -A and stop the IPA service.
+        3. Check the authentication again.
+        4. Check the feedback console message
+    :expectedresults:
+        1. User authenticates successfully and gets the TGT.
+        2. Successfully remove the TGT and IPA is not reachable.
+        3. User authenticate successfully, did not get TGT of user
+        4. Get 'No Kerberos TGT granted as the server does not support this method.
+           Your single-sign on(SSO) experience will be affected' console message as IPA service is stopped.
+    :customerscenario: False
+    """
+    with open(f"{testdatadir}/passkey-mapping.ipa") as f:
+        ipa.user("user1").add(user_auth_type="passkey").passkey_add(f.read().strip())
+
+    client.sssd.start()
+
+    rc, _, output, _ = client.auth.su.passkey_with_output(
+        username="user1",
+        pin=123456,
+        device=f"{moduledatadir}/umockdev.device",
+        ioctl=f"{moduledatadir}/umockdev.ioctl",
+        script=f"{testdatadir}/umockdev.script.ipa",
+        command="kdestroy -A",
```

Comment:
I'd prefer to run this command in a separate code line to make it obvious to the reader that you are destroying the ticket. Indeed, I'd run klist here and check that the ticket has been obtained, and then remove it.

Reply:
The way we have written the [code](https://github.com/SSSD/sssd-test-framework/blob/master/sssd_test_framework/utils/authentication.py#L384) I have those options:

1. Pass `command = "klist; kdestroy -A"` check the output for getting tgt and destroy it.
2. Use same with two times, first pass the `klist` as a command the pass `kdestory -A` as  a command

Reply:
@ikerexxe The challenge here is that if specified in another line, it would be a different SSH session. IIRC, the command is appended and added to the expect script maintaining the same SSH session. 

Alternatively, it could be written like this? I'm not sure if it will work. 
```
    with client.ssh('tuser', 'Secret123') as ssh:
        with client.auth.kerberos(ssh) as krb:
            result = krb.klist()
            assert f'krbtgt/{kdc.realm}@{kdc.realm}' in result.stdout
```
https://tests.sssd.io/en/latest/guides/testing-ldap-krb5.html

Reply:
ok, thank you for the explanation. I withdraw this comment.